notfound/utils.py

```diff
@@ -64,9 +64,8 @@ def replace_uris(app, doctree, nodetype, nodeattr):
                 imagedir=imagedir,
             )
         else:
-            uri = '/{language}/{version}/{imagedir}{filename}'.format(
-                language=app.config.notfound_default_language,
-                version=app.config.notfound_default_version,
+            uri = '{prefix}{imagedir}{filename}'.format(
```

Comment:
I already know you're going to hit issues with a trailing `/`, so this logic should account for a prefix with and without a trailing `/`:

```suggestion
            uri = '{prefix.rstrip('/')}/{imagedir}{filename}'.format(
```

Reply:
wouldn't be better to validate the `prefix` set by the user has a beginning _and_ trailing `/` and show a warning? I'm not sure if it will be sane to support it in the code, since we are going to hit this random issues.

Maybe it's safer to let the user know about this and make them to fix the config instead. What do you think?

Reply:
That seems fine to me also. We should define exactly what we expect though.

Reply:
Agree with @humitos here, warning is a sane check here. Less magic in code - more you can configure without code edit.

Reply:
I updated the PR with a change for this.